include/openssl/ssl.h.in

```diff
@@ -316,158 +316,135 @@ typedef int (*SSL_verify_cb)(int preverify_ok, X509_STORE_CTX *x509_ctx);
 /* Typedef for SSL async callback */
 typedef int (*SSL_async_callback_fn)(SSL *s, void *arg);
 
-/* Disable Extended master secret */
-# define SSL_OP_NO_EXTENDED_MASTER_SECRET                0x00000001U
 
-/* Cleanse plaintext copies of data delivered to the application */
+/*
+ * SSL/TLS Connection Options.
+ */
+    /* Disable Extended master secret */
+# define SSL_OP_NO_EXTENDED_MASTER_SECRET                0x00000001U
+    /* Cleanse plaintext copies of data delivered to the application */
 # define SSL_OP_CLEANSE_PLAINTEXT                        0x00000002U
-
-/* Allow initial connection to servers that don't support RI */
+    /* Allow initial connection to servers that don't support RI */
 # define SSL_OP_LEGACY_SERVER_CONNECT                    0x00000004U
-
-/* Enable support for Kernel TLS */
+    /* Enable support for Kernel TLS */
 # define SSL_OP_ENABLE_KTLS                              0x00000008U
-
 # define SSL_OP_TLSEXT_PADDING                           0x00000010U
+# define SSL_OP__UNUXED                                  0x00000020U
 # define SSL_OP_SAFARI_ECDHE_ECDSA_BUG                   0x00000040U
 # define SSL_OP_IGNORE_UNEXPECTED_EOF                    0x00000080U
-
+    /* See below SSL_OP_ALLOW_CLIENT_RENEGOTIATION           0x00000100U */
```

Comment:
Why is this one out of order? I think it should be defined here in this list

Reply:
I thought it more useful to keep related options together, but the current `0x000....U` syntax makes that hard. That's why I'd like to use the `1<<xxx` syntax. But that's discussed in #15145 so I moved it.